package.json

```diff
@@ -18,7 +18,7 @@
     "build-test-app-local": "yarn workspace teams-test-app build:local",
     "clean": "lerna run clean --stream",
     "clean:full": "yarn clean && yarn clean:modules",
-    "clean:modules": "lerna clean -y && if exist node_modules (rd /s /q node_modules)",
+    "clean:modules": "lerna clean -y && rm -rf node_modules",
```

Comment:
Sadly I was too optimistic about it working on Windows. My test case was atypical and it looks like it won't work for all Windows developers. Going to use `rimraf` instead, as per a Stack Overflow article (thanks Aaron!). Stay tuned for the update.

Reply:
Fixed.